app/components/Footer.tsx

```diff
@@ -0,0 +1,102 @@
+import {
+  Box,
+  Heading,
+  Grid,
+  GridItem,
+  Link,
+  LinkProps,
+  Stack,
+  Text,
+} from "@chakra-ui/react";
+import Image from "next/image";
+
+const links = [
+  { href: "/about-us", text: "About Us" },
+  { href: "/view-machines", text: "View Machines" },
+  { href: "/careers", text: "Careers" },
+  { href: "/request-quote", text: "Contact Us" },
+  { href: "/request-quote", text: "Request a Quote" },
+];
+
+const contactInfo = [
+  {
+    heading: "Mailing Address",
+    content: [
+      "14751 North Kelsey Street, Suite 105",
+      "PMB 616",
+      "Monroe, WA 98272-1458",
+    ],
+  },
+  {
+    heading: "Shipping Address",
+    content: ["30330 NE 172nd Street", "Duvall, WA 98019"],
+  },
+  {
+    heading: "Phone",
+    content: ["(425) 788-2100"],
+  },
+  {
+    heading: "Email",
+    content: ["sales@kenworthymachine.com"],
+  },
+];
+
+const Footer = () => {
+  return (
+    <Grid
+      templateColumns="repeat(5, 1fr)"
+      p={{ md: "42px 54px", lg: "72px 63px" }}
+    >
+      <GridItem colSpan={2}>
+        <Stack spacing={{ md: "50px", lg: "50px" }}>
+          <Box>
+            <Image
+              src="/images/kenworthy-logo.jpg"
+              alt="Kenworthy Machine"
+              width={500}
+              height={500}
```

Comment:
The logo image looks kind of large on my macbook screen (13"). I tried to add the `layout="responsive"` property to the Image component, but it's just making the image larger on larger screens. Maybe that property combined with adjusting the width and height?
For context: The layout prop in Next.js's Image component sets how the image responds to the container size. This property means the image will scale based on the width of the parent element while maintaining its aspect ratio determined by the width and height properties that you set.

Reply:
<img width="1437" alt="Screenshot 2024-01-25 at 11 22 16 AM" src="https://github.com/wavecrestweb/kenworthy-machine/assets/115492619/09fbcef9-6b12-448a-b635-e76663498382">

Reply:
@avinashi10 Thanks for pointing this out. I didn't set up logo resizing for different screens. Will read about Next.js Image component a bit more and will adjust it. But, I don't see `layout` prop in the list of props for Image component, maybe that's why it didn't do anything? Again, will read more about it and address it later today.

Reply:
I think the logo is correct or close to the correct size according to the design (it is wider in the design: 523px!). But, it does scale down in size according to screen width in the Footer, whereas in the Header, it doesn't. I can't tell why that is. We might want to consult Kayla and see which way would be better.

Reply:
> @avinashi10 I don't see `layout` prop in the list of props for Image component. 
I'm reviewing your PR now and Next.js helpfully gives us a warning in the console for the hero-photo that `layout` prop is a legacy prop.